README.md

```diff
@@ -1 +1,14 @@
-# awesome-list
\ No newline at end of file
+# Awesome Projects [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
+
+> A curated list of open source projects from students at KPI
+
+## List
+
+- [nano_daw](https://github.com/abelkevich/nano_daw)
+
+## License
+
+[![CC0](https://mirrors.creativecommons.org/presskit/buttons/88x31/svg/cc-zero.svg)](https://creativecommons.org/publicdomain/zero/1.0)
+
+To the extent possible under law, Oleksii Belkevych has waived all copyright and
```

Comment:
Should it be just your name or 
```suggestion
To the extent possible under law, Oleksii Belkevych and contributors have waived all copyright and
```

Reply:
WDYT?

Reply:
I don't know much about copyright aspects, but your change fits much better